### PR TITLE
Responsive in progress

### PR DESCRIPTION
**Pull request by devarshidoshi**
## Description
Responsive part covered up to testimonial

## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

File: twindle-web/style.css [hunk 1]
```diff
@@ -158,10 +236,21 @@ main .email_subs {
   padding: 2rem 4rem;
 }
 
-main .email_subs .email_input {
```
Comment (Akshay2996) [inline twindle-web/style.css:161]: Please can you explain removing .email_input result to what?

File: twindle-web/style.css [hunk 2]
```diff
@@ -52,7 +50,7 @@ body::-webkit-scrollbar-thumb{
 /* Navigation Bar */
 
 header {
-  display: flex;
+  display: inline;
```
Comment (Akshay2996) [inline twindle-web/style.css:53]: Why did you remove the display: flex property, & used display:inline. Please also explain this too what it result too. Flexbox is a great way to make the header responsive

File: twindle-web/style.css [hunk 3]
```diff
@@ -79,12 +78,8 @@ header a {
   align-items: center;
 }
 
-header nav {
-  flex: 1;
-}
-
 header nav ul {
-  display: flex;
+  display: block;
```
Comment (Akshay2996) [inline twindle-web/style.css:82]: Here too you removed flex to block?

**Review by Akshay2996 — CHANGES_REQUESTED** [review]
I have made some comments in the code. Please explain to me why it was changed so that I can understand it better 🙂

**Review by PuruVJ — APPROVED** [review]

File: twindle-web/style.css (hunk 1 above)
Comment (devarshidoshi) [reply twindle-web/style.css:161]: This is because i need to give different width to input according to view width

File: twindle-web/style.css (hunk 2 above)
Comment (devarshidoshi) [reply twindle-web/style.css:53]: Capture

File: twindle-web/style.css (hunk 3 above)
Comment (devarshidoshi) [reply twindle-web/style.css:82]: afterwards i have made that code comment because making this code responsive was creating many difficulties

**Review by Akshay2996 — APPROVED** [review]
@proful Please merge this PR 🙂
